src/sass/components/_EditsNav.scss

```diff
@@ -78,11 +80,12 @@
         }
         .step {
           background-color: #dc731c;
-          background-image: url("/img/exclamation.png");
-          background-size: 4px auto;
-          background-position: center;
-          background-repeat: no-repeat;
+          background-image: none;
           border-color: #dc731c;
+          color: #fff;
+          font-size: 30px;
+          line-height: 25px;
+          padding: 0;
```

Comment:
Changing this to `0 1px 0 0` seems to center align it better.

There is also something different between the `!` in the nav and the `!` when its used in the alert, but I can't see what would be causing it.